airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -964,6 +964,9 @@ class BigQueryCreateExternalTableOperator(BaseOperator):
     :param schema_object: If set, a GCS object path pointing to a .json file that
         contains the schema for the table. (templated)
     :param source_format: File format of the data.
+    :param autodetect: Try to detect schema and format options automatically.
```

Comment:
Shouldn't we use `auto_detect`?
I see that in the hook it's also `autodetect` :thinking:

Reply:
Thanks for the review. I went with `autodetect` because the hook and [bq command-line tool](https://cloud.google.com/bigquery/docs/schema-detect) that form.

Reply:
ah I guess I'm wrong on this anyway as autodetect is one word :)
So you are correct!